git-repo-manager/src/service.rs

```diff
@@ -55,18 +54,29 @@ impl GitRepoManager for GitRepoManagerService {
             return Err(Status::unknown("Failed initializing repository"));
         }
 
-        // add post-receive hook : le lien symbolique peut être remplacé
-        if Command::new("sh")
-            .arg("-c")
-            .arg(format!(
-                "ln -s {}/hooks/post-receive {}/hooks/post-receive",
-                self.config.githook_base_path.clone(),
-                new_repo_path.clone(),
+        let hook_path = format!(
+            "{}/hooks/post-receive",
```

Comment:
I think it should rather be "{}/post-receive", that way we can put the post-receive anywhere without having to add hooks, what do you think ?